manifests/addons/values-prometheus.yaml

```diff
@@ -13,7 +13,7 @@ server:
   # Use port 9090 to match Istio documentation
   service:
     servicePort: 9090
-
+  readinessProbeInitialDelay: 0
```

Comment:
The default is 30s which is super slow for tests (and users!). our current install uses 0 so this seems reasonable (and is k8s default)